catalog-items.html

```diff
@@ -4,13 +4,16 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>SKISIRS#4587 - BinaBox</title>
-    <link rel="stylesheet" href="./css/style.css">    
-</head>
+    <link rel="stylesheet" type="text/css" href="https://necolas.github.io/normalize.css">
```

Comment:
It's not working - if you open this link you can see:
<img width="869" alt="Screenshot 2024-08-10 at 23 34 19" src="https://github.com/user-attachments/assets/1dfedd5c-f6da-4002-a176-aca3d22ce528">

You need to push download button and copy all code to separate file in your css directory